client/editSchedule.js

```diff
@@ -66,22 +33,35 @@ function getScheduledRounds(competitionId) {
       title: 1,
     }
   }).fetch();
-  return rounds;
-}
-function getUnscheduledRounds(competitionId) {
-  var rounds = Rounds.find({
-    competitionId: competitionId,
-    startMinutes: {$exists: false}
-  }, {
+  // Round that are scheduled for times outside of the range shown by our calendar
+  // are considered unscheduled.
+  var competition = Competitions.findOne(competitionId, {
     fields: {
-      eventCode: 1,
-      roundCode: 1,
-
-      durationMinutes: 1,
-      nthDay: 1,
-      title: 1,
+      numberOfDays: 1,
+      calendarStartMinutes: 1,
+      calendarEndMinutes: 1,
     }
-  }).fetch();
+  });
+  function roundIsScheduled(round) {
```

Comment:
Looking it up in the function is the clean design. You could worry about the performance of loading the same Competition again and again, but I assume/hope/pray that we have a reasonable Mongo query cache somewhere in this stack. And even if we don't, it is only at most 3 extra queries, which I assume is < 10ms.

You could also pass in the Competition to the method to optimize performance, but I don't like that as much.